src/plugins/shared_ux/public/plugin.ts

```diff
@@ -16,6 +17,7 @@ import {
 
 import { servicesFactory } from './services';
 
+let services: SharedUxServices;
```

Comment:
This is an anti-pattern.  We should not be keeping a module-level instance.  Can you tell me why this is necessary?